db/python/tables/sequence.py

```diff
@@ -219,6 +223,7 @@ async def get_sequence_ids_for_sample_id(
         Get the sequence IDs from internal sample_id and sequence type
         Map them to be keyed on sequence type
         """
+        # TODO: how should we address this, because we ultimately want to split on the technology I think
```

Comment:
Do you think a nested dict would be too clunky?

Reply:
I think sequence groups would solve this completely, so I'll keep the behaviour here the same (no breaking changes), and then more seriously action sequence groups.